pkg/test-infra/fixture/mysql.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fixture
+
+// MySQLConfig is the configuration for a MySQL component.
+// TODO: enable binlog
```

Comment:
I see there is an `EnableBinlog` option now. Do we still need this todo?

Reply:
I forgot to remove it, it's my fault. removed in https://github.com/pingcap/tipocket/pull/220/commits/01a18e6682c11bb4ab2b1cec1eea349304badf81.